app/services/article-comments.js

```diff
@@ -108,11 +115,20 @@ export default Service.extend({
         }));
       };
 
+      const notify = (message, type, timeout = 5000) => {
+        this.wdsBannerNotifications.addNotification({
```

Comment:
The notification does not appear sticky (to the top of the viewport). That should be fixed.